src/navbar/navbar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+
+
+const Navbar = props => {
+    return (
+    <nav class="navbar is-transparent">
+                <div class="navbar-brand">
+                    <a class="navbar-item" href="https://bulma.io">
+                        <img src="https://bulma.io/images/bulma-logo.png" alt="Bulma: a modern CSS framework based on Flexbox" width="112" height="28" /> </a>
+
+                </div>
+
+                    <div id="navbarExampleTransparentExample" class="navbar-menu">
+                        <div class="navbar-start">
+                        <a class="navbar-item" href="https://bulma.io/"> Home</a>
+                        </div>
+            
+                            <div class="navbar-end">
+                                <div class="navbar-item">
+                                    <div class="field is-grouped ">

```

Comment:
```suggestion
                    <div id="navbarExampleTransparentExample" className="navbar-menu">
                        <div className="navbar-start">
                        <a className="navbar-item" href="https://bulma.io/"> Home</a>
                        </div>
            
                            <div className="navbar-end">
                                <div className="navbar-item">
                                    <div className="field is-grouped ">

```
Para especificar una clase CSS, usa el atributo className.